### ISSUE TITLE
Denon mode: ability to operate with channel faders < 100%

### ISSUE THREAD

**Issue by wrightlaw** [erikrichardlarson/unbox#42]
Fantastic work on this - one feature request would make it perfect for me: Would it be possible for it to have an option to pick up track changes where channel faders are < 100%? I generally mix with the crossfader and generally have channel faders at < 100% during normal operation.

Thank you!

**Comment by erikrichardlarson** [comment]
Yeah this is coming in the next update, the plan was to grab the channel with the highest fader position and report that out, will this work for you?

**Comment by wrightlaw** [comment]
I think that'll do the job, from memory moving the crossfader position alters the reported channel values? e.g. so if channel1 and 2 were faded up 75% and the crossfader was faded fully to the left, you'd get channel1 reported at 75% and channel2 reported at 0%? This would then result in channel1 "winning" in unbox and its trackname being shown.

If that holds true, that'd be lovely for me :)
